roles/openshift_node/defaults/main.yml

```diff
@@ -113,6 +113,7 @@ system_images_registry_dict:
   origin: "docker.io"
 
 system_images_registry: "{{ system_images_registry_dict[openshift_deployment_type | default('origin')] }}"
+l_openshift_use_external_openvswitch: "{{ openshift_use_external_openvswitch | default(false) }}"
```

Comment:
We don't need to use the l_ prefix and indirect here.  This variable is only going to apply to the node role.

Just use
```yaml
openshift_use_external_openvswitch: False
```

Reply:
Thanks for the review, will change! :D